app/webpacker/components/ContactsPage/ContactForm.jsx

```diff
@@ -102,17 +79,12 @@ export default function ContactForm({ userDetails }) {
               name="contactRecipient"
               value={contactRecipient}
               checked={selectedContactRecipient === contactRecipient}
-              onChange={setSelectedContactRecipient}
+              onChange={() => dispatch(updateContactRecipient(contactRecipient))}
```

Comment:
Since this is already inside a `map` block, the `contactRecipient` in this scope is the actual recipient that needs to be changed to. Hence, didn't use the parameter from onChange.

Reply:
This may "accidentally" work, but it's not conceptually correct for reactive components.
You should always use the value "closest" to the update function.

Reply:
Okay done